2021-12-27 16:52:52,911 INFO    MainThread:12460 [wandb_setup.py:_flush():71] setting env: {}
2021-12-27 16:52:52,911 INFO    MainThread:12460 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-27 16:52:52,911 INFO    MainThread:12460 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-27 16:52:52,911 INFO    MainThread:12460 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20211227_165252-254m4itx\logs\debug.log
2021-12-27 16:52:52,911 INFO    MainThread:12460 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20211227_165252-254m4itx\logs\debug-internal.log
2021-12-27 16:52:52,912 INFO    MainThread:12460 [wandb_init.py:init():404] calling init triggers
2021-12-27 16:52:52,912 INFO    MainThread:12460 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-27 16:52:52,912 INFO    MainThread:12460 [wandb_init.py:init():449] starting backend
2021-12-27 16:52:52,912 INFO    MainThread:12460 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-27 16:52:52,917 INFO    MainThread:12460 [backend.py:ensure_launched():199] starting backend process...
2021-12-27 16:52:53,214 INFO    MainThread:12460 [backend.py:ensure_launched():204] started backend process with pid: 14936
2021-12-27 16:52:53,215 INFO    MainThread:12460 [wandb_init.py:init():458] backend started and connected
2021-12-27 16:52:53,215 INFO    MainThread:12460 [wandb_init.py:init():519] updated telemetry
2021-12-27 16:52:53,223 INFO    MainThread:12460 [wandb_init.py:init():549] communicating current version
2021-12-27 16:52:54,510 INFO    MainThread:12460 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-27 16:52:54,510 INFO    MainThread:12460 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-27 16:52:54,715 INFO    MainThread:12460 [wandb_init.py:init():592] starting run threads in backend
2021-12-27 16:52:55,646 INFO    MainThread:12460 [wandb_run.py:_console_start():1816] atexit reg
2021-12-27 16:52:55,647 INFO    MainThread:12460 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-27 16:52:55,647 INFO    MainThread:12460 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-27 16:52:55,649 INFO    MainThread:12460 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-27 16:52:55,649 INFO    MainThread:12460 [wandb_init.py:init():619] run started, returning control to user process
2021-12-27 16:52:56,718 INFO    MainThread:12460 [wandb_run.py:finish():1463] finishing run austinbergstrom/places_dry_run/254m4itx
2021-12-27 16:52:56,719 INFO    MainThread:12460 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-27 16:52:56,719 INFO    MainThread:12460 [wandb_run.py:_restore():1758] restore
2021-12-27 16:52:57,659 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 647
  total_bytes: 647
}

2021-12-27 16:52:58,211 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 647
  total_bytes: 647
}

2021-12-27 16:52:58,394 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 647
  total_bytes: 647
}

2021-12-27 16:52:58,728 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 647
  total_bytes: 4256
}

2021-12-27 16:52:58,829 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 647
  total_bytes: 4256
}

2021-12-27 16:52:58,930 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 647
  total_bytes: 4256
}

2021-12-27 16:52:59,035 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4256
  total_bytes: 4256
}

2021-12-27 16:52:59,141 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4256
  total_bytes: 4256
}

2021-12-27 16:52:59,246 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4256
  total_bytes: 4256
}

2021-12-27 16:52:59,349 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4256
  total_bytes: 4256
}

2021-12-27 16:52:59,454 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4256
  total_bytes: 4256
}

2021-12-27 16:52:59,783 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4256
  total_bytes: 4256
}

2021-12-27 16:53:00,249 INFO    MainThread:12460 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4256
  total_bytes: 4256
}
local_info {
}

2021-12-27 16:53:01,375 INFO    MainThread:12460 [wandb_run.py:_append_files():2184] logging synced files
